cdi/src/main/java/org/togglz/cdi/spi/CDIBeanFinder.java

```diff
@@ -21,6 +21,7 @@ public class CDIBeanFinder implements BeanFinder {
     public final static String BEAN_MANAGER_JNDI_TOMCAT = "java:comp/env/BeanManager";
 
     public final static String SERVLET_CONTEXT_ATTR_WELD_1_1 = "org.jboss.weld.environment.servlet.javax.enterprise.inject.spi.BeanManager";
+    public final static String SERVLET_CONTEXT_ATTR_WELD_5_1 = "org.jboss.weld.environment.servlet.jakarta.enterprise.inject.spi.BeanManager";
```

Comment:
I'd suggest reordering access modifiers here and for existing constants as well to comply with the Java Language specification (`public static final`)